src/biip/__init__.py

```diff
@@ -55,10 +55,11 @@
     from importlib_metadata import version, PackageNotFoundError  # type: ignore
 
 from biip._exceptions import BiipException, EncodeError, ParseError
-from biip._parser import ParseResult, parse
+from biip._parser import ParseResult, parse, parse_hri
 
 __all__ = [
     "parse",
+    "parse_hri",
```

Comment:
This change must be undone now with the `parse_hri` function moved.

Reply:
Done. Will let you review and resolve these threads.